ocaml/idl/datamodel.ml

```diff
@@ -1200,11 +1200,20 @@ let _ =
     ~doc:"The requested update could not be found. Please upload the update again. This can occur when you run xe update-pool-clean before xe update-apply. " ();
   error Api_errors.update_pool_apply_failed [ "hosts" ]
     ~doc:"The update cannot be applied for the following host(s)." ();
+  error Api_errors.update_apply_failed [ "output" ]
+    ~doc:"The update apply failed. Please see attached output." ();
```

Comment:
Have we discusses this previously? A better phrase would be `The updated failed to apply` or `The application of the update failed.`